library/src/main/kotlin/jp/co/panpanini/FileGenerator.kt

```diff
@@ -21,6 +21,8 @@ class FileGenerator {
         val packageName = file.kotlinPackageName ?: return Pair(emptyList(), typeMappings)
         val files = file.types.map { type ->
             FileSpec.builder(packageName, type.kotlinTypeName)
+                    .addComment("Code generated by protok protocol buffer plugin, do not edit.\n")
+                    .addComment("Source file: ${type.name}.proto")
```

Comment:
This is incorrect - the types could be inside the same file, but will be generated as multiple models